src/controller/user.py

```
@@ -199,7 +199,7 @@ async def process_deposit_ctrl(session: SessionDep, deposit: DepositDTO) -> Proc
 
 async def register_oldboy_applicant_ctrl(session: SessionDep, user: User) -> OldboyApplicant:
     user_created_at_aware = user.created_at.replace(tzinfo=timezone.utc)
-    if datetime.now(timezone.utc) - user_created_at_aware < timedelta(weeks=52 * 3): raise HTTPException(400, detail="must have been a member for at least 3 years.")
+    if datetime.now(timezone.utc) - user_created_at_aware < timedelta(weeks=52 * 3):raise HTTPException(400, detail="must have been a member for at least 3 years.")
```

Ruff (0.13.3)

Multiple statements on one line (colon)

(E701)